### PR TITLE
Sleeves

### PR DESCRIPTION
**Pull request by EvilMelonTheSecond**
## About The Pull Request
Adds option to roll up sleeves halfway to many uniforms

## Why It's Good For The Game
Style options good. Probably. Sprites provided by AbsoluteJerry

I've almost made it before the new year as a gift. oh well

## Changelog
:cl:
add: Added half rolled sleeves to marine uniforms
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/under.dm [hunk 1]
```diff
@@ -64,9 +64,10 @@
 		ATTACHMENT_SLOT_UNIFORM_TIE,
 		ATTACHMENT_SLOT_BADGE,
 	)
-	///Typepath list of allowed attachment types.
+	///Typepath list of uniform variants.
 	var/list/adjustment_variants = list(
 		"Down" = "_d",
+		"Half" = "_h",
```
Comment (hyper2snyper) [inline code/modules/clothing/under/under.dm:70]: This is adding a halve version of sleeves to ALL the uniforms. Unless you have sprites for them all, you should overwrite on the specific ones you have sprites for.

**Review by hyper2snyper — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/marine_uniform.dm [hunk 2]
```diff
@@ -17,6 +21,7 @@
 /obj/item/clothing/under/marine/camo
 	name = "\improper TGMC camo fatigues (jungle)"
 	icon_state = "m_marine_jumpsuit"
+	adjustment_variants = list()
```
Comment (hyper2snyper) [inline code/modules/clothing/under/marine_uniform.dm:24]: why are you removing the variants for these?

File: code/modules/clothing/under/marine_uniform.dm [hunk 3]
```diff
@@ -46,6 +51,7 @@
 	adjustment_variants = list()
 	has_sensor = 2
 	flags_item_map_variant = null
+	adjustment_variants = list()
```
Comment (hyper2snyper) [inline code/modules/clothing/under/marine_uniform.dm:54]: this is overwriting the `adjustment_variants` right above it.
look two lines up.

**Review by hyper2snyper — CHANGES_REQUESTED** [review]

**Comment by EvilMelonTheSecond** [comment]
I should check my own code to spare the sanity of maintainers, this PR is too simple to have been corrected two times.

**Review by Kuro020 — CHANGES_REQUESTED** [review]
isnt this missing some of the marine jumpsuits?

**Comment by EvilMelonTheSecond** [comment]
> isnt this missing some of the marine jumpsuits?

They are. The sprites aren't mine I was just the one to implement them

**Pull request merged**